instrumentation/github.com/gin-gonic/gin/otelgin/config.go

```diff
@@ -115,6 +115,8 @@ func WithMeterProvider(mp metric.MeterProvider) Option {
 
 // WithMetricAttributeFn specifies a function that extracts additional attributes from the http.Request
 // and returns them as a slice of attribute.KeyValue.
+//
+// If attributes are duplicated between this method and `WithGinMetricAttributeFn`, the attributes in this method will be used.
```

Comment:
Well, in this case, it's the opposite!

```suggestion
// If attributes are duplicated between this method and `WithGinMetricAttributeFn`, the attributes in this method will be overriden.
```